Main.cpp

```diff
@@ -149,6 +171,14 @@ int main(int argc, char** argv) {
     return 1;
   }
 
+  if (should_check_config) {
+    auto ret = parseAndCompile(flag_conf_file);
+    if (!ret) {
+      std::cerr << "Logging failed to uncompilable config.\n";
```

Comment:
```suggestion
      OLOG << "Config is not valid";
```
Prefer `OLOG` since logging is already initialized. Also this message should be more accurate.